### DISCUSSION TITLE
独立悬浮窗版 志琼插件 安装不了的解决方案

### DISCUSSION THREAD

**Discussion by dawnlyn** [YuehaiTeam/cocogoat#127] [General]
**原因**：证书到期
**解决方案**：修改系统时间为 **`2022/4/15`** 到 **`2023/4/16`** 之间即可

# **记得安装完把时间改回来！**

**Comment by liuahfy** [comment]
楼主，安装之后如何自动追踪定位啊。我这边打开悬浮窗只有米游社地图，左下角插件不显示了，也没有任何提示去下载插件
